Maquetado/index.html

```diff
@@ -0,0 +1,151 @@
+        </span>
+      </div>
+
+      <div class="techno">
+        <img src="assets/elm_icon.svg" alt= "Elm" class="logo-techn">
+        <h3 class="techno-title"> Elm </h3>
+        <h6 class="description" >
+          Release: 2016
+        </h6>
+        <span>Elm is a domain-specific functional programming language for declaratively creating web browser-based graphical user interfaces.</span>
+      </div>
+
+      <div class="techno">
+        <img src="assets/polymer_icon.svg" alt= "Polymer" class="logo-techn">
+        <h3 class="techno-title">
+          Polymer
+        </h3>
+        <h6 class="description" >
+          Release: 2015
+        </h6>
+        <span>
+          Polymer is an open-source JavaScript library for building web applications using Web Components.
+        </span>
+      </div>
+    </div>
+  </div>
+
+  <div class="container-techno">
+    <h1 class="type-tool">
+      Back-End Tools
+    </h1>
+
+   <div class="tools-container">
+    <div class="techno">
+      <img src="assets/rails_icon.png" alt= "Ruby"  class="logo-techn" >
+      <h3 class="techno-title">
+        Ruby on Rails
+      </h3>
+      <h6 class="description" >
+        Release: 2005
+      </h6>
+      <span>
+        Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.
+      </span>
+    </div>
+
+    <div class="techno">
+      <img src="assets/express_icon.svg" alt= "Express" class="logo-techn">
+      <h3 class="techno-title">
+        NodeJS
+      </h3>
+      <h6 class="description" >Release: 2009
+      </h6>
+      <span>
+        Node.js is a JavaScript runtime built on Chromes's V8 JavaScript engine. Uses an event-driven, non-blocking I/O model that makes it lightweight and efficient.
+      </span>
+    </div>
+
+    <div class="techno">
+      <img src="assets/aspnet_icon.png" alt="VueJS" class="logo-techn">
+      <h3 class="techno-title">
+        ASP.NET Core
+      </h3>
+      <h6 class= "description" >
+        Release: 2016
+      </h6>
+      <span>
+        ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the ull .NET framework, on Windows, and the cross-platform .NET Core.
+      </span>
+    </div>
+
+    <div class="techno">
+      <img src= assets/django_icon.svg alt= "Elm" class="logo-techn">
+      <h3 class="techno-title">
+        Django
+      </h3>
+      <h6 class= "description" > Release: 2005  </h6>
+      <span>Django (/dʒæŋɡoʊ/ JANG-goh) is a free and open source web framework, written in Python, which follows the model-view-template (MVT) architectural pattern. </span>
+    </div>
+   </div>
+  </div>
+</div>
+
+<footer class="footer">
+  <form class="form-style">
+   <h1 class="footer-font">
+     Contact us!
+   </h1>
+   <input class="imput" type="text" name="email">
```

Comment:
I think this is a `type="email"`, also the class should be something like `email-input` or something like that. More self explanatory :smile: